proposals/helm-repo-container-registry-convergence/007-helm-store-cache.md

```diff
@@ -0,0 +1,14 @@
+# Proposal 7 - Helm Chart Cache
```

Comment:
I think that this topic is valid regardless of which direction we go as it relates to Docker convergence.

We should remove this and open it as its own proposal. People would benefit from this.

Please also see related issue https://github.com/helm/community/issues/54

Reply:
there are several items here. Happy to split up and converge with other proposals, like #54 as it makes sense. Perhaps to discuss on the call today to figure out specific next steps.